File: genetic-packer/src/main/java/genetic/packer/GeneticPackerApplication.java
package genetic.packer;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = "genetic")
public class GeneticPackerApplication {

	public static void main(String[] args) {
		SpringApplication.run(GeneticPackerApplication.class, args);
	}
}

File: genetic-packer/src/main/java/genetic/packer/GeneticPackerApplication.kt
package genetic.packer

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["genetic"])
class GeneticPackerApplication

fun main(args: Array<String>) {
  runApplication<GeneticPackerApplication>(*args)
}